### PR TITLE
fix sxp error message

### PR DESCRIPTION
**Pull request by samrickman**
PR to solve this [issue](https://github.com/r-lib/lobstr/issues/66) with the error message for the `lobstr::sxp()` function.

### CODE REVIEW

File: R/sxp.R [hunk 1]
```diff
@@ -52,13 +52,18 @@
...
+    abort((
+      paste(
+        "`expand` must contain only values from:",
+        paste0("'", opts, "'", collapse = ", ")
+      )
+    ))
   }
 
-  obj_inspect_(x,
+  obj_inspect_(
```
Comment (hadley) [inline R/sxp.R:65]: Would you mind not restyling in this PR? It's better to fix code style in a separate PR so it's easy to focus on the substantive changes.

**Review by hadley — COMMENTED** [review]
Thanks! Do you also want to add a test with `expect_snasphot(error = TRUE)`?

File: R/sxp.R (hunk 1 above)
Comment (samrickman) [reply R/sxp.R:65]: Yes I thought the same thing after I saw it on github - I'll fix it and add a test.

Comment (samrickman) [reply R/sxp.R]: @hadley I have added the test and removed the restyling (format on save was the culprit!). Hope this does the job now - if not let me know.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**